### PR TITLE
[docs] README typo fixes

### PR DESCRIPTION
**Pull request by sarah11918**
## Change

- two small typo fixes to the Cloudflare integration README ("you" -> "your" and a missing "to")

## Testing

No tests, docs typo fixes

## Docs

only docs! README fixes.

### CODE REVIEW

File: packages/integrations/cloudflare/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Cloudflare Pages has 2 different modes for deploying functions, `advanced` mode
...
 
-In directory mode the adaptor will compile the client side part of you app the same way, but moves the worker script into a `functions` folder in the project root. The adaptor will only ever place a `[[path]].js` in that folder, allowing you to add additional plugins and pages middleware which can be checked into version control. Cloudflare documentation contains more information about [writing custom functions](https://developers.cloudflare.com/pages/platform/functions/).
+In directory mode the adaptor will compile the client side part of your app the same way, but moves the worker script into a `functions` folder in the project root. The adaptor will only ever place a `[[path]].js` in that folder, allowing you to add additional plugins and pages middleware which can be checked into version control. Cloudflare documentation contains more information about [writing custom functions](https://developers.cloudflare.com/pages/platform/functions/).
```
Comment (delucis) [inline packages/integrations/cloudflare/README.md:52]: Don't we also usually spell adapter with an e?

```suggestion
In directory mode the adapter will compile the client side part of your app the same way, but moves the worker script into a `functions` folder in the project root. The adapter will only ever place a `[[path]].js` in that folder, allowing you to add additional plugins and pages middleware which can be checked into version control. Cloudflare documentation contains more information about [writing custom functions](https://developers.cloudflare.com/pages/platform/functions/).
```

**Review by delucis — COMMENTED** [review]
Looks good! Left one comment about spelling.

File: packages/integrations/cloudflare/README.md (hunk 1 above)
Comment (sarah11918) [reply packages/integrations/cloudflare/README.md:52]: I feel like maybe this page should be edited properly. I came to fix one thing someone spotted... spotted another myself... 😅

Comment (sarah11918) [reply packages/integrations/cloudflare/README.md:52]: (There's one more "adaptor" in the paragraph above this too. 😅 )

File: packages/integrations/cloudflare/README.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Cloudflare Pages has 2 different modes for deploying functions, `advanced` mode
 
 For most projects the adaptor default of `advanced` will be sufficient; the `dist` folder will contain your compiled project. Switching to directory mode allows you to use [pages plugins](https://developers.cloudflare.com/pages/platform/functions/plugins/) such as [Sentry](https://developers.cloudflare.com/pages/platform/functions/plugins/sentry/) or write custom code to enable logging.
```
Comment (sarah11918) [inline packages/integrations/cloudflare/README.md:50]:
```suggestion
For most projects the adapter default of `advanced` will be sufficient; the `dist` folder will contain your compiled project. Switching to directory mode allows you to use [pages plugins](https://developers.cloudflare.com/pages/platform/functions/plugins/) such as [Sentry](https://developers.cloudflare.com/pages/platform/functions/plugins/sentry/) or write custom code to enable logging.
```

**Pull request merged**